Corzbank.Web/ClientApp/src/app/data/models/confirmation.model.ts

```diff
@@ -1,4 +1,10 @@
+import { Guid } from "guid-typescript";
+import { VerificationType } from "../enums/verificationType.enum";
+
 export class ConfirmationModel {
-  email: string;
+  email?: string;
```

Comment:
should not be nullable. String can contain null itself

Reply:
resolved